frictionless/eia860.py

```diff
@@ -11,26 +11,10 @@
     "description":
         "US Energy Information Administration (EIA) Form 860 data for electric power "
         "plants with 1 megawatt or greater combined nameplate capacity.",
```

Comment:
The descriptions of the sources do not exist in `pudl.metadata.constants`. Should we add them?

Reply:
Yeah.

Reply:
These have been integrated into the `pudl.metadata.sources.SOURCES` dictionary now.